docs/resources/apig_acl_policy_associate.md

```diff
@@ -0,0 +1,54 @@
+---
+subcategory: "API Gateway (Dedicated APIG)"
+---
+
+# huaweicloud_apig_acl_policy_associate
+
+Use this resource to bind the APIs to the ACL policy within HuaweiCloud.
+
+## Example Usage
+
+```hcl
+variable "instance_id" {}
+variable "policy_id" {}
+variable "api_publish_ids" {
+  type = list(string)
+}
+
+resource "huaweicloud_apig_acl_policy_associate" "test" {
+  instance_id = var.instance_id
+  policy_id   = var.policy_id
+  publish_ids = var.api_publish_ids
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String, ForceNew) Specifies the region where the ACL policy and the APIs are located.  
+  If omitted, the provider-level region will be used. Changing this will create a new resource.
+
+* `instance_id` - (Required, String, ForceNew) Specifies the ID of the dedicated instance to which the APIs and the
+  ACL policy belong.  
+  Changing this will create a new resource.
+
+* `policy_id` - (Required, String, ForceNew) Specifies the ACL Policy ID for APIs binding.  
```

Comment:
looks only one apig_acl_policy_associate can be created for each policy_id, if so, please add a note

Reply:
done.